1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 10 Line 5_
```markdown
### The Shell

A shell is a program that executes commands from the user and displays the result. There are many different shell programs - bash is quite popular; it’s been around since 1989 and is the default on macOS and most Linux distributions. 

![shell_image](img/1-terminal-shell.png)
```
When I'm back at my desktop, I can replace this with a windows/anaconda screenshot.